Cargo.toml

```diff
@@ -39,5 +39,11 @@ pre-release-replacements = [
 cargo-emit = "0.2.1"
 displaydoc = { version = "0.2", default-features = false }
 lazy_static = "1.4"
+mockall_double = "0.2.1"
```

Comment:
I think this can be a `dev-dependency`...

Reply:
Will change